test/rest-api/response/body/body-stream.mocks.ts

```diff
@@ -0,0 +1,24 @@
+import { setupWorker, rest } from 'msw'
+
+const stream = new ReadableStream({
+  start(controller) {
+    controller.enqueue('line1\n')
+
+    setTimeout(() => {
+      controller.enqueue('line2\n')
+      controller.close()
+    }, 0)
+  },
+})
+
+const worker = setupWorker(
+  rest.get('/sse', (req, res, ctx) => {
+    return res(
+      ctx.status(200),
+      ctx.set('Content-Type', 'text/plain'),
```

Comment:
I believe that such a response must also specify that its body is a stream.

Reply:
apparently not needed for a stream of `text/plain`